browser/hub/internal/android/java/java_sources.gni

```diff
@@ -3,9 +3,5 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
-# Since we make bytecode changes on the same component and extend it at the same time, we
-# have to patch it directly to avoid circular dependency.
-brave_browser_hub_internal_java_sources = [
-  "//brave/browser/hub/internal/android/java/src/org/chromium/chrome/browser/hub/BraveHubManagerImpl.java",
-  "//brave/browser/hub/internal/android/java/src/org/chromium/chrome/browser/hub/BraveHubToolbarView.java",
-]
+# Unsuccessfull attempt to get rid brave_browser_hub_internal_java_sources
+brave_browser_hub_internal_java_deps = [ "//brave/browser/hub/internal:java" ]
```

Comment:
it's not necessary for `chrome/browser/hub/internal` to depend on `brave/browser/hub/internal` because the override is done after the bytecode is generated. `//brave/browser/hub/internal:java` should just be added to `brave_chrome_java_deps`

Reply:
thanks lot @bridiver , I didn't see the possibility to use `brave_chrome_java_deps`, addressed the notice at 
https://github.com/brave/brave-core/pull/28679/commits/2796d9bb8afe0220f2637d7b04b7872644627b9c